scripts/pkg-config-wrapper.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# On Windows, static libraries are using "foo.lib" naming.
+# On Unix, they are in the form of "libfoo.a"
+# On Windows, "foo.lib" can also the definition of symbols in a foo.dll.
+# So you can link to "foo.lib", whatever you are doing static or dynamic linking and you are good.
+# However, meson is always creating static library as "libfoo.a"
+# 'to avoid a potential name clash with shared libraries which also generate import libraries with a lib suffix.' [1]
+# On top of that, qmake is replacing all `-lfoo` in LIBS by `foo.lib` (on Windows).
+# So at the end, we try to link with `foo.lib` but we have `libfoo.a`
+# Solution could be :
+# - Rename `libfoo.a` to `foo.lib`, but it would mean modify deps libraries on the FS
+# - Don't use LIBS and direction set QMAKE_LFLAGS but we would have to handle different command line option format
```

Comment:
direction → directly (also fix in the commit message)

Reply:
Done in the fixup commit.
Commit message already fixed